header.html

```diff
@@ -1,10 +1,87 @@
-  <div class="i5k-login">
-    <a href="https://i5k.nal.usda.gov/user/login">Login</a>
-  </div>
+<link rel="stylesheet" href="css/HeaderFooter.css" media="screen">
+<script src="js/jquery.js"></script><script src="js/hoverIntent.js"></script><script src="js/superfish.min.js"></script><script>(function($){$(document).ready(function(){var example = $("#example").superfish({});});})(jQuery);</script>
+<div class="container">
+   <div class="header">
+      <div id="sign-lockup"><a class="logo title="United States Department of Agriculture"><img src="../header/usda-logo.svg" alt="Home"></a>
+         <a class="name" title="United States Department of Agriculture"><span class="usda-name">United States Department of Agriculture</span>National Agricultural Library</a>
+      </div>
+      <div class="site-title">i5k Workspace@NAL</div>
+   </div>
+   <ul class="sf-menu sf-js-enabled sf-arrows" id="example">
+      <li><a href='https://i5k.nal.usda.gov/'>Home</a></li>
+      <li class="has-sub">
+         <a href='https://i5k.nal.usda.gov/species'>Organisms</a>
+      </li>
+      <li class="has-sub">
+         <a href='https://i5k.nal.usda.gov/'>Data</a>
+         <ul>
+            <li><a href='https://i5k.nal.usda.gov/content/data-downloads'>Data Downloads</a></li>
+            <li class="has-sub">
+               <a href='https://i5k.nal.usda.gov/data-submission-overview'>Submit Data</a>
+               <ul>
+                  <li><a href='https://i5k.nal.usda.gov/datasets/submit-a-dataset'>Submit a dataset</a></li>
+                  <li><a href='https://i5k.nal.usda.gov/data-submission-overview'>Data Submission Overview</a></li>
+                  <li><a href='https://i5k.nal.usda.gov/datasets/request-project'>Request a new i5k Workspace Project</a></li>
+                  <li class="has-sub"><a href='https://i5k.nal.usda.gov/image_permission/new'>Sign an Image Permission Form</a></li>
+               </ul>
+            </li>
+         </ul>
+      </li>
+      <li class="has-sub">
+         <a href='https://i5k.nal.usda.gov/'>Tools</a>
+         <ul>
+            <li><a href='https://i5k.nal.usda.gov/blast'>BLAST</a></li>
+            <li class="has-sub">
+               <a href='https://i5k.nal.usda.gov/'>JBrowse/Apollo</a>
+               <ul>
+                  <li><a href='https://i5k.nal.usda.gov/web-apollo-registration'>Apollo registration form</a></li>
+                  <li><a href='https://i5k.nal.usda.gov/available-genome-browsers'>JBrowse/Apollo Organisms</a></li>
+               </ul>
+            </li>
+            <li><a href='https://i5k.nal.usda.gov/legacy_blast'>Legacy BLAST</a></li>
+            <li><a href='https://i5k.nal.usda.gov/https://i5k.nal.usda.gov/webapp/hmmer/'>HMMER (beta)</a></li>
+            <li><a href='https://i5k.nal.usda.gov/https://i5k.nal.usda.gov/webapp/clustal'>Clustal (beta)</a></li>
+            <li class="has-sub">
+               <a href='https://i5k.nal.usda.gov/'>Training tools</a>
```

Comment:
It would be good if we can use double quotes rather than single quotes when setting the HTML attributes.